src/main/java/de/t0biii/music/main/Main.java

```diff
@@ -81,4 +91,43 @@ public void Updater() {
       }
     }
   }
+  
+  public void reloadConfigs() {
+	  this.reloadConfig();
+	  cm.loadConfig();
+	  this.saveConfig();
+	  this.loadUserConfigsIfNeeded();
+  }
+  
+  public void loadUserConfigsIfNeeded() {
+	if (this.getConfig().getBoolean("options.allowDisabling")) {
+	  createUserConfigs();
+	}
+  }
+  
+  private void createUserConfigs() {
+	File userConfigsFile = new File(getDataFolder(), "disabledPlayers.yml");
+    if (!userConfigsFile.exists()) {
+      userConfigsFile.getParentFile().mkdirs();
+      saveResource("disabledPlayers.yml", false);
+    }
+	userConfigs = new YamlConfiguration();
+	try {
+        userConfigs.load(userConfigsFile);
+    } catch (IOException | InvalidConfigurationException e) {
+        e.printStackTrace();
+    }
+  }
+  
+  public FileConfiguration getUserConfigs() {
+    return this.userConfigs;
+  }
+  
+  public void saveUserConfigs() {
+	try {
+	  this.userConfigs.save(new File(getDataFolder(), "disabledPlayers.yml"));
+	} catch (IOException e) {
+		e.printStackTrace();
+	}
+  }
```

Comment:
we can but in in a own File i think

Reply:
I moved it into ConfigManager (I think it makes sense there) but if you want I can make another file for it.